File: eval/rule_accuracy/corpus/java/correctness/tp_correctness.java
// Correctness issues — true positives

public class Correctness {
    public Object badCatchAll() {
        try {
            return riskyOperation();
        } catch (Exception e) {  // expect: java/java-catch-exception
            return null;  // expect: java/java-null-return
        }
    }
}

File: eval/rule_accuracy/corpus/rust/correctness/tp_correctness.rs
// Correctness issues — true positives

fn bad_clone_loop(items: &[String]) -> Vec<String> {
    let mut result = Vec::new();
    for item in items {
        result.push(item.clone());  // expect: rust/rs-clone-in-loop
    }
    result
}

fn bad_todo() {
    todo!("implement this");  // expect: rust/rs-todo-macro
}
